### ISSUE TITLE
add olemeta  for the use in Python applications

### ISSUE THREAD

**Issue by duese22** [decalage2/oletools#131]
Hello,
do i get it right that it is currently not possible to use olemeta within own python applications?

**Comment by decalage2** [comment]
You are right. Actually, olemeta is just a thin CLI wrapper calling olefile.OleFileIO.get_metadata. So you can simply use that method from olefile in your own python apps:

- http://olefile.readthedocs.io/en/latest/Howto.html#extract-metadata
- http://olefile.readthedocs.io/en/latest/olefile.html#olefile.OleMetadata
